core/dbt/parser/read_files.py

```diff
@@ -49,17 +70,17 @@ def load_source_file(
             skip_loading_schema_file = True
 
     if not skip_loading_schema_file:
-        file_contents = load_file_contents(path.absolute_path, strip=False)
-        source_file.checksum = FileHash.from_contents(file_contents)
-        source_file.contents = file_contents.strip()
+        # We strip the file_contents before generating the checksum because we want
```

Comment:
This is a change in behavior. It felt really wrong that we couldn't actually generate the checksum from the stored file contents, since it was done on the file contents prior to the contents being stripped and stored.

The alternative would have been to store the unstripped version of the contents.

Reply:
Makes sense to me.